base_layer/wallet/src/transaction_service/protocols/transaction_broadcast_protocol.rs

```diff
@@ -205,31 +215,36 @@ where TBackend: TransactionBackend + Clone + 'static
                         self.timeout = to;
                         info!(
                             target: LOG_TARGET,
-                            "Broadcast monitoring protocol (Id: {}) timeout updated to {:?}", self.id ,self.timeout
+                            "Transaction monitoring event 'updated_timeout' triggered (Id: {}), timeout updated to {:?}", self.id ,self.timeout
+                        );
+                    } else {
+                        trace!(
```

Comment:
Is this not an error? Perhaps add a warning here?

Reply:
Na, it occurs if somehow this worker has lagged behind the pubsub buffer which can happen if there is a lot going on and the Send process above takes a while.